Source/FullScreenPokemon.d.ts

```diff
@@ -2076,6 +2076,16 @@ declare module FullScreenPokemon {
     }
 
     /**
+     * 
+     */
+    export interface ICyclingRoad extends IDetector {
```

Comment:
`ICyclingRoad` isn't an extremely accurate name. The convention is to mention what it does, not the original-game feature/area name.